app/views/devices_guidance/about_the_offer.md

```diff
@@ -13,32 +13,16 @@ This offer applies to maintained schools, pupil referral units, academy trusts a
 
-### Find out [how we’ve calculated how many devices you can order](/devices/allocation-and-specification).
-
-Devices can be ordered for disadvantaged children in years 3 to 11 when:
-
-* a local health protection team has advised a group of children (such as a ‘bubble’ or year group) not to attend school
-* a primary school is only open to vulnerable children and the children of critical workers ([tier 4](https://www.gov.uk/government/publications/containing-and-managing-local-coronavirus-covid-19-outbreaks/covid-19-contain-framework-a-guide-for-local-decision-makers#tier-4))
-* a secondary school or college is only open to vulnerable children and the children of critical workers ([tier 3](https://www.gov.uk/government/publications/containing-and-managing-local-coronavirus-covid-19-outbreaks/covid-19-contain-framework-a-guide-for-local-decision-makers#tier-3) or [4](https://www.gov.uk/government/publications/containing-and-managing-local-coronavirus-covid-19-outbreaks/covid-19-contain-framework-a-guide-for-local-decision-makers#tier-4)) 
-* a school or college is fully open, but supporting a disadvantaged child living in another area who is unable to attend due to local travel restrictions
-
-Devices can be ordered for disadvantaged children in any year group who have been advised to shield when:
-
-* a school or college supports a [clinically extremely vulnerable child who is shielding](https://www.gov.uk/government/publications/guidance-on-shielding-and-protecting-extremely-vulnerable-persons-from-covid-19/guidance-on-shielding-and-protecting-extremely-vulnerable-persons-from-covid-19) following official or medical advice
-* a school or college supports a child who has been advised to shield as a member of their household is clinically extremely vulnerable
-
-Devices cannot be ordered when:
-
-* a secondary school or college is limiting attendance by operating a rota model ([tier 2](https://www.gov.uk/government/publications/containing-and-managing-local-coronavirus-covid-19-outbreaks/covid-19-contain-framework-a-guide-for-local-decision-makers#tier-2))
-* disadvantaged children are [self-isolating because of coronavirus symptoms](https://www.gov.uk/government/publications/covid-19-stay-at-home-guidance/stay-at-home-guidance-for-households-with-possible-coronavirus-covid-19-infection) where this is not part of broader recommendations to a ‘bubble’ or year group
-* disadvantaged children are [self-isolating following arrival in the UK](https://www.gov.uk/government/publications/coronavirus-covid-19-how-to-self-isolate-when-you-travel-to-the-uk/coronavirus-covid-19-how-to-self-isolate-when-you-travel-to-the-uk)
+#### Find out [how we’ve calculated how many devices you can order](/devices/allocation-and-specification).
```

Comment:
Should this still be a heading if there's no content beneath it?

Reply:
I think H4 has been used here to make the CTA stand out rather than for its proper structural purpose. Since it's important this goes live now, I've approved it but if there's a better way we should do this here please let us know and we'll update and republish asap. @fofr @Lloyd-K